qa/tasks/cephfs/test_strays.py

```diff
@@ -817,6 +817,8 @@ def test_dirfrag_limit(self):
                 for n in range(0, {file_count}):
                     with open(os.path.join(path, "%s" % n), 'w') as f:
                         f.write(str(n))
+                dfd = os.open(path, os.O_DIRECTORY)
+                os.fsync(dfd) # for asynchronous creates on kclient
```

Comment:
looks like the code to create a dir with contents and fsync is repeated 5 times here. Can we have a helper instead, something similar to create_n_files in mount.py?

Reply:
Ya, will fix this and some other lceanup.